srtcp.go

```diff
@@ -59,9 +63,7 @@ func (c *Context) decryptRTCP(dst, encrypted []byte) ([]byte, error) {
 		}
 	}
 
-	out := allocateIfMismatch(dst, encrypted)
```

Comment:
note: `allocateIfMismatch` handled nil `dst` but `growBuffer` in `cipher.decryptRTCP` also handles nil output buffer